jekyll/_cci2/testing-orbs.md

```diff
@@ -19,224 +19,268 @@ This guide covers various best practices for testing orbs.
-parameters:
-  to:
-    type: string
-    default: "World"
-    description: "Hello to whom?"
-steps:
-  - run:
-      environment:
-        PARAM_TO: <<parameters.to>>
-      name: Hello Greeting
-      command: <<include(scripts/greet.sh)>>
+```shell
+shellcheck src/scripts/*.sh
+```
 
+Or, using CircleCI's Local Execute:
+```shell
+circleci local execute --job shellcheck/check
 ```
 
-{:.tab.unitTest.greet-sh}
+### Review
+{: #review }
 
-```shell
-# Source: https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/src/scripts/greet.sh
+The orb-tools orb includes a job `orb-tools/review` which will run a suite of tests against your orb designed to find opportunities to implement best practices and improve the quality of the orb. The "review" job was modeled closely after _ShellCheck_, and operates based on a list of rules called "RC" Review Checks. Each "RC" code corresponds to a specific rule, which can optionally be ignored.
 
-Greet() {
-    echo Hello "${PARAM_TO}"
-}
+Review Checks output to JUNIT XML formatted and are automatically uploaded to CircleCI to be displayed natively in the UI.
 
-# Will not run if sourced for bats-core tests.
-# View src/tests for more information.
-ORB_TEST_ENV="bats-core"
-if [ "${0#*$ORB_TEST_ENV}" == "$0" ]; then
-    Greet
-fi
+![orb-tools review check RC008]({{site.baseurl}}/assets/img/docs/orbtools-rc008.png)
 
-```
+When you click into the error you will receive more information such as what file and at what line in the code the error was found, along with suggestions for resolution.
+
+#### Local Review
+
+The "review" job is built using the [BATS-Core](https://github.com/bats-core/bats-core) bash automation testing framework. If you have bats installed locally, you can review your orb by running:
 
-{:.tab.unitTest.greet-bats}
 ```shell
-# Source: https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/src/tests/greet.bats
-
-# Runs prior to every test
-setup() {
-    # Load our script file.
-    source ./src/scripts/greet.sh
-}
-
-@test '1: Greet the world' {
-    # Mock environment variables or functions by exporting them (after the script has been sourced)
-    export PARAM_TO="World"
-    # Capture the output of our "Greet" function
-    result=$(Greet)
-    [ "$result" == "Hello World" ]
-}
+bats src/scripts/review.bats
+```
 
+Or, using CircleCI's Local Execute:
+```shell
+circleci local execute --job orb-tools/review
 ```
 
-### BATS Core
-{: #bats-core }
+## Unit testing
+{: #unit-testing }
 
-The [Bash Automation Testing System](https://github.com/bats-core/bats-core) is an open source testing framework that provides a simple way to test UNIX programs.
+If you are taking advantage of the orb development kit's [`<<include(file)>>` file inclusion]({{site.baseurl}}/2.0/orb-concepts/#file-include-syntax) feature and `src/scripts` directory to store and source your bash files, you can write true integration tests for your scripts.
```

Comment:
Have "orb development kit" with upper-case initials to keep it consistent with the rest of the docs.